src/Sentry/Laravel/Integration.php

```diff
@@ -82,4 +84,30 @@ public static function setTransaction($transaction): void
     {
         self::$transaction = $transaction;
     }
+
+    /**
+     * Block until all async events are processed for the HTTP transport.
+     *
+     * @internal This is not part of the public API and is here temporarily until
+     *  the underlying issue can be resolved, this method will be removed.
+     */
+    public static function flushEvents(): void
+    {
+        $client = Hub::getCurrent()->getClient();
+
+        if ($client instanceof Client) {
```

Comment:
We should implement `close`/`flush` in the main SDK, this should make this obsolete.
This code should go directly in there.

Reply:
@HazAT yes, we had extensive discussion about it and the consensus is we made a mistake implementing the async stuff, so this is a workaround until we can fix that in the base SDK 👍 this will be removed at a later time since it won't be needed if fixed in the base SDK.